scripts/setup_gemdaq.sh

```diff
@@ -106,6 +143,22 @@ else
   return 1
 fi
```

Comment:
Lines 112-144 would benefit from refactoring. Suggestion:
```bash
if [[ $SYSTEM_INFO == *"lxplus"* ]];
then
    # LCG 93 doesn't provide `virtualenv' in PATH
    VIRTUALENV="python -m virtualenv"
    PIP="python -m pip"
    if [[ $KERNEL_VERSION == *"2.6."* ]];
    then
        OS_VERSION="slc6"
        source /cvmfs/sft.cern.ch/lcg/views/LCG_93/x86_64-slc6-gcc7-opt/setup.sh
    elif [[ $KERNEL_VERSION == *"3.10."* ]];
    then
        OS_VERSION="cc7"
        source /cvmfs/sft.cern.ch/lcg/views/LCG_93/x86_64-centos7-gcc7-opt/setup.sh
    else
        echo "Unrecognized kernel version! Exiting..."
        return 1
    fi
elif [[ "$DNS_INFO" == "cms" ]];
then
    # We are in the .cms network
    WGET="ssh cmsusr wget"
 fi
```

Reply:
You forgot the 904 machines, that can be either `slc6` or `cc7`

Reply:
well, add it ;) Anyway, it should be refactored in a way that lines like 131-137 do not repeat.

Reply:
Just make the 
```bash
if [[ $KERNEL_VERSION == *"2.6."* ]];
...
```
the outmost condition